site/src/_pages/docs/4_faq.mdx

```diff
@@ -252,7 +252,7 @@ All schemas created on the Block Protocol website are currently public. In futur
 
 <FAQ question="Doesn't a decentralized schema registry result in problems?">
 
-Moving beyond a single set of constrained schemas that rarely change introduces new issues. We believe that all of these issues can be mitigated and that the benefits of an open ecosystem of well-described, discoverable types grealy outweigh the drawbacks.
+Moving beyond a single set of constrained schemas that rarely change introduces a number of issues. However, we believe that all of these can be mitigated and that the benefits of an open ecosystem of well-described, discoverable, and end-user defineable types greatly outweighs any potential drawbacks.
```

Comment:
Same comment about 'defineable'

Also I think "outweigh" was correct as you're referring to "benefit_s_"? I'm unsure

i.e.
"the benefits outweigh"
as opposed to
"the benefits outweighs"

Reply:
Addressed in https://github.com/blockprotocol/blockprotocol/commit/fa28251b8df96c64802a0ea8208d98d1a4abc056 - ty for spotting the typo introduced in the previous typo fix commit 🙃

Reply:
`definable` -> `definable` fixed in https://github.com/blockprotocol/blockprotocol/commit/90953b8cbf5266d7599abbc08fbf87aec679d724